### ISSUE TITLE
Parent container not expanding to show Images in iOS

### ISSUE THREAD

**Issue by williambuchanan2** [dotnet/maui#19175]
### Description

If you put images into a collection view, the container does not expand to accommodate the image. So you just end up seeing the images as a slit.

### Steps to Reproduce

Open demo application from reproduction repository.
Tap "Image Error"
Observe as below - images are not shown in full size.

<img width="405" alt="Screenshot 2023-12-03 at 4 52 31 pm" src="https://github.com/dotnet/maui/assets/32165641/303b89db-b6c6-493e-b6a9-d4bd0befc244">

### Link to public reproduction project repository

https://github.com/williambuchanan2/MauiNavigation

### Version with bug

8.0.3

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Did you find any workaround?

No

**Comment by samhouts** [comment]
Was this working in .NET 7?

**Comment by williambuchanan2** [comment]
Just wondering if this is going to be treated as a priority. It is fairly fundamental functionality so would be good to get it fixed sooner rather than later.

**Comment by williambuchanan2** [comment]
Any update on this? I think this is the root cause of the scroll jitter which has been reported on iphone so it's causing havoc.

**Comment by williambuchanan2** [comment]
Still no update on this critical and long running problem?

**Comment by acaliaro** [comment]
@samhouts news about this?

**Comment by HarishwaranVijayakumar** [comment]
The issue is not reproducible in the latest .NET MAUI version 9.0.100 on iOS 18.0 in both CV1 and CV2.

https://github.com/user-attachments/assets/53ca2c30-35e5-4126-b71c-4295fb0ddd85

**Comment by williambuchanan2** [comment]
Yes this issue is now fixed. Had checked recently and it wasn't so must have been a recent change.
